src/components/Preview/Preview.tsx

```diff
@@ -117,11 +118,16 @@ export const Preview = ({
 
   const audioEnabled = useHMSStore(selectIsLocalAudioEnabled);
   const videoEnabled = useHMSStore(selectIsLocalVideoDisplayEnabled);
+  const localPeerRole = useHMSStore(selectLocalPeerRole);
+  const audioAllowed =
```

Comment:
selectIsAllowedToPublish use this selector instead to get audio/video allowed permission

Reply:
Done